package.json

```diff
@@ -67,10 +67,12 @@
     "react-helmet": "^6.0.0",
     "react-json-view": "^1.19.1",
     "react-lazylog": "4.5.2",
+    "react-markdown": "^5.0.2",
     "react-query": "2.4.14",
     "react-router-dom": "5.1.2",
     "react-scripts": "3.4.1",
     "reflexbox": "^4.0.6",
+    "remark-gfm": "^1.0.0",
```

Comment:
Same comment as other PR, is this needed?

Reply:
ah yeah - that was one of the plugins I was using before I borrowed our css overrides from catalog. Just pushed up a commit that removes it